src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+      key: D
+      order: 3
+- uuid: db6081e9-bc7a-4430-af57-cccd291dec2d
+  type: SingleChoice
+  content: Conversion of material assets into digital assets using DeFi is known as.....
+  hint: NoHint
+  explanation: Tokenization means conversion of material assets into digital tokens using DeFi and smart contracts.
+  answerKeys:
+    - B
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: Digitization
+      key: A
+      order: 0
+    - content: Tokenization
+      key: B
+      order: 1
+    - content: Consensus Mechanism
+      key: C
+      order: 2
+    - content: None of the these
+      key: D
+      order: 3
+- uuid: 66f8098a-3b77-458f-a7f8-6c951bd19343
+  type: MultipleChoice
+  content: How does tokenization empower DeFi?
+  hint: Benefits of Tokenization
+  explanation: Tokens make it easy to trade material assets in the form of tokens. These tokens are extremely secured and cannot be stolen or hacked.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: Material assets could be converted.
+      key: A
+      order: 0
+    - content: Lending and borrowing become easy.
+      key: B
+      order: 1
+    - content: Forgery of assets increases.
+      key: C
+      order: 2
+    - content: Assets become more secured.
+      key: D
+      order: 3
+- uuid: 945a2feb-81db-46bb-8e33-9dcf95ce7307
+  type: SingleChoice
+  content:  What is the major issue involved in Smart Contracts?
+  hint: NoHint
+  explanation: Smart contracts are complicated currently. The tiniest flaw in their code can lead to loss funds.
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Smart contracts are not trustworthy.
+      key: A
+      order: 0
+    - content: It is difficult to code smart contracts.
+      key: B
+      order: 1
+    - content: The slightest flaw in code can lead to a loss of funds.
+      key: C
+      order: 2
+    - content: They are easy to hack.
+      key: D
+      order: 3
+- uuid: df4273d6-ea88-45b0-90f1-c28d496dbb6e
+  type: SingleChoice
+  content: What is the current number of transactions that Ethereum can process per second?
+  hint: NoHint
+  explanation: Ethereum can currently process 13 transactions per second. This number is bound to increase in future with advancements in the technology.
+  answerKeys:
+    - D
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: 1000
+      key: A
+      order: 0
+    - content: 110
+      key: B
+      order: 1
+    - content: 10,000
+      key: C
+      order: 2
+    - content: 13
+      key: D
+      order: 3
+- uuid: 163f7e64-856c-4238-890a-93afe3e3481a
+  type: SingleChoice
+  content: As Decentralised finance is a new technology it is quite uncertain.
```

Comment:
We should have max of 2-3 true or false questions per chapter.

Reply:
Changed the question